cmd/updater/subcommands/bootstrap/command.go

```diff
@@ -32,6 +38,35 @@ type cliParams struct {
 	version string
 }
 
+const (
+	installScriptParamsFile = "/etc/datadog-installer.ini"
```

Comment:
why isn't this in /etc/datadog/datadog-installer.ini?

Reply:
Both are fine, figured it would cause less permission issues to not put it in a directory since it's a one-off

Reply:
It won't be a problem as the installer is in the agent group and has read/write access to /etc/datadog/

Reply:
I think it's cleaner to have all in `/etc/datadog` less new components